src/components/sequencing/SequenceEditor.svelte

```diff
@@ -41,11 +38,13 @@
   import CommandPanel from './CommandPanel/CommandPanel.svelte';
 
   export let actionsWithSequenceParameters: ActionDefinition[] = [];
-  export let channelDictionary: ChannelDictionary | null = null;
-  export let commandDictionary: CommandDictionary | null = null;
+  export let phoenixContext: PhoenixContext = {
```

Comment:
I would avoid making variables that frequently get updated a single object. This is because updating a portion of the object would require the entire object's pointer to change (i.e. cloned) in order to properly trigger svelte to rerender.